### ISSUE TITLE
Bower package?

### ISSUE THREAD

**Issue by brophdawg11** [balderdashy/merge-defaults#4]
Great little extension to lodash!  Any chance you could add it to the bower registry?

**Comment by masterspambot** [comment]
Already available on bower as I mentioned in #5.

**Comment by brophdawg11** [comment]
But that's as an npm-style module.exports package, and thus requires browserify, which I'm not using.  I'm shimming it right now onto the lodashAngularWrapper bower package: https://gist.github.com/brophdawg11/0e8e3fc77777d9579dd1

**Comment by sailsbot** [comment]
Thanks for posting, @brophdawg11.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
